File: src/main/java/classes/Coordinates.java
package classes;

/**
 * Entity coordinate class
 */
public class Coordinates {
    private Float x; //Поле не может быть null
    private Integer y; //Поле не может быть null

    public Coordinates(Float x, Integer y){
        this.x = x;
        this.y = y;
    }

    public String toString() {
        return "(" + this.x + ", " + this.y + ")";
    }

}

File: common/src/main/kotlin/com/github/diosa/dms/collection/Coordinates.kt
package com.github.diosa.dms.collection

import io.github.landgrafhomyak.itmo.dms_lab.interop.DisplayName
import kotlinx.serialization.Serializable

/**
 * Entity coordinate class
 */
@Serializable
class Coordinates (
    @DisplayName("Координата х")
    val x: Float, //Поле не может быть null
    @DisplayName("Координата у")
    val y: Int    //Поле не может быть null
)